package.json

```diff
@@ -37,7 +36,6 @@
     "jszip": "^3.4.0",
     "mime": "^2.4.4",
     "plyr": "^3.5.10",
-    "rdflib": "^0.19.1",
```

Comment:
Is this dependency not being used anymore?

Reply:
Couldn't find any reference to it and tested the app functionally after removal